microsoft-edge/devtools-guide-chromium/speed/get-started.md

```diff
@@ -22,440 +22,491 @@ ms.date: 05/04/2021
+    The rest of the `map-on-demand.js` file is the same as the `map.js` file. As soon as the Bing Map API loads, the `GetMap` function runs. `GetMap` configures and displays the map in the container element.
 
-### Eliminate render-blocking resources
+1. Save your changes in Visual Studio Code, then refresh the webpage in Microsoft Edge, and run a new audit in the **Lighthouse** tool to see how your changes affect the load performance:
 
-Your latest report says that eliminating render-blocking resources is now the biggest opportunity.
+    ![The Lighthouse tool report showing a better score after the changes](./get-started-images/lighthouse-report-without-map.png)
 
-A render-blocking resource is an external JavaScript or CSS file that the browser must download, parse, and run before it displays the page.  The goal is to only run the core CSS and JavaScript code that is required to display the page properly.
+    Your **Lighthouse** score is now at 31, which is an improvement from before, but there are more things to optimize.
+
+#### Eliminate render-blocking resources
+
+The next opportunity displayed in the **Opportunities** section of the **Lighthouse** tool is related to eliminating render-blocking resources.
+
+A render-blocking resource is an external JavaScript or CSS file that the browser must download, parse, and run before it displays the page. The more render-blocking resources need to be processed when a webpage loads, the longer the webpage needs, to start appearing in the browser. Try to only run the core CSS and JavaScript code that's required to display the initial state of the page properly.
 
 The first task, then, is to find code that you don't need to run on page load.
 
-1. Click **Eliminate render-blocking resources** to display the resources that are blocking: `lodash.js` and `jquery.js`.
+1. Click **Eliminate render-blocking resources** to display the resources that are blocking:
 
-   ![More information about the Eliminate render-blocking resources opportunity](../media/speed-glitch-tony-remix-updated-audits-performance-oppportunities-expanded.msft.png)
+   ![More information about the Eliminate render-blocking resources opportunity](./get-started-images/render-blocking-css.png)
 
-1. Press `Ctrl`+`Shift`+`P` (Windows, Linux) or `Command`+`Shift`+`P` (macOS) to open the Command Menu, start typing `Coverage`, and then select **Show Coverage**.
+    **Lighthouse** displays a list of the stylesheets that the demo webpage uses, such as: `base.css`, `home.css`, and `map.css`.
 
-   ![Open the Command Menu from the Audits panel](../media/speed-glitch-tony-remix-updated-audits-performance-oppportunities-expanded-command-coverage.msft.png)
+1. Open the **Coverage** tool again: press `Ctrl`+`Shift`+`P` (Windows, Linux) or `Command`+`Shift`+`P` (macOS), type `Coverage`, and then select **Show Coverage**.
```

Comment:
```suggestion
1. Open the **Coverage** tool again: press `Ctrl`+`Shift`+`P` (Windows, Linux) or `Command`+`Shift`+`P` (macOS), type **Coverage**, and then select **Show Coverage**.
```
previous style research and PRs in this repo concluded that entering text in GUI UI context rather than code editor generally uses bold rather than fixed-width.